lib/plug/upload.ex

```diff
@@ -144,7 +144,7 @@ defmodule Plug.Upload do
   end
 
   @doc false
-  def start_link() do
+  def start_link(_) do
```

Comment:
We should probably do this for backwards compatibility:

```suggestion
  def start_link(_ \\ []) do
```

Reply:
Actually, ignore this. :)

Reply:
sorry, clicked accept too quickly